test/gst-msdk/encode/10bit/vp9.py

```diff
@@ -0,0 +1,103 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from ...util import *
+from ..encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+class VP9_10EncoderTest(EncoderTest):
+  def before(self):
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "msdkvp9enc",
+      gstdecoder    = "ivfparse ! msdkvp9dec hardware=true",
```

Comment:
Please see #160... ivfparse is no longer correct.

Reply:
You may use ivfparse if you use avmux_ivf in the command line for encoding but avmux_ivf is available with the latest gst-libav, so you have to build the latest gst-libav first when running  your test cases.